cmd/api/src/api/v2/file_uploads_integration_test.go

```diff
@@ -130,6 +130,16 @@ func Test_FileUpload(t *testing.T) {
 		assert.Nil(tx, err)
 		assert.Equal(tx, http.StatusUnsupportedMediaType, resp.StatusCode)
 	})
+
+	t.Run("not valid json", func(tx *testing.T) {
+		jsonInput := loader.GetReader("v6/ingest/SO_Logo.png")
```

Comment:
I opted to toss an image in here to validate. Wasn't sure what image to use so went with logo, suggestions are appreciated!

Reply:
Seems fine